test/test_benchmark.py

```diff
@@ -112,10 +113,33 @@ def display(self, baseline=None):
 
     # size
 
-    size = sum(os.path.getsize(f) for f in self.get_output_files())
-    gzip_size = sum(len(zlib.compress(read_binary(f))) for f in self.get_output_files())
+    def add_measurement(name, size, gzip_size):
+      self.size_measurements.append({
+        'value': name,
+        'measurement': size,
+      })
+      self.size_measurements.append({
+        'value': name + ' (gzipped)',
+        'measurement': gzip_size,
+      })
+
+    total_size = 0
+    total_gzip_size = 0
+    save_measurements = hasattr(self, 'size_measurements') and self.size_measurements is not None
+
+    for file in self.get_output_files():
+      size = os.path.getsize(file)
+      gzip_size = len(zlib.compress(read_binary(file)))
+      if save_measurements:
+        # Remove the 'size_' prefix added by the benchmark runner.
```

Comment:
I don't understand... why does every file start with `size_`?

Reply:
When each benchmarker builds it prefixes all the output files with the name of the benchmarker.